docker-compose.yml

```diff
@@ -1,21 +1,12 @@
 version: '3.7'
 services:
-  db:
```

Comment:
It is change related to the local dev environment. It is not necessary to keep the old database container.